herbert_core/classes/MPIFramework/@parallel_config/private/check_and_set_cluster_.m

```diff
@@ -11,8 +11,10 @@
 mff = MPI_clusters_factory.instance();
 
 if isempty(wrkr)
-    error('HERBERT:parallel_config:not_available',...
+    warning('HERBERT:parallel_config:not_available',...
         'Parallel worker is not on the Matlab path so parallel features are not available')
+    config_store.instance().store_config(obj,...
+        'parallel_cluster','none','cluster_config','none');
```

Comment:
@mducle @abuts Hi Duc, Thanks for all the diagnostic work you put in here. I am wondering if this is the best approach. Horace emits warnings left right and centre and it is easy to ignore them. What will the state of execution be if the warning is emitted and the code carries on? My feeling is that there should be 2 possibilities. If the user has not selected to run parallel, then a warning should be acceptable. But if the user has selected parallel then the run should error. Ideally the initialisation should not be wrong in the first place. What do you think?

Reply:
Sorry... forgot about this PR.

> What will the state of execution be if the warning is emitted and the code carries on?

It will proceed without using the cluster (e.g. with only shared-memory / thread parallelisation).

The main issue here is the hard-coding on the types of "machines" (e.g. `win_large` / `idaaas_large`) to automatically using clustering even though the cluster worker is not defined - the user needs to set-up paths for the cluster workers so this cannot be automatic...

If we don't put the check here then we should put it in the code which detects which type of machine it is / or the code which sets up what options each type of machine - to check if the worker script has been set up...

I was really hoping that @abuts would sort this out really...